qbatch

```diff
@@ -83,7 +103,12 @@ if __name__ == "__main__":
 
     # compute the number of jobs needed. This will be the number of elements in
     # the array job
-    num_jobs = int(math.ceil(len(task_list) / float(chunk_size)))
+    if system == 'local':
+        use_array = False
+        num_jobs = 1
+        chunk_size= sys.maxsize
```

Comment:
I guess `num_jobs` and `chunk_size` as configured here assume we don't have more than `sys.maxsize` commands. `sys.maxsize` should be enough commands for anyone... ;-)